ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__amd64-nightly.yaml

```diff
@@ -1900,6 +1900,27 @@ tests:
     test:
     - ref: cucushift-hypershift-extended-health-check
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: azure-ipi-ovn-hypershift-no-marketplace-guest-f14
+  cron: 15 21 15,29 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      HYPERSHIFT_AZURE_MARKETPLACE_ENABLED: "false"
```

Comment:
If you add this env var, should you need update all the existing aks hypershift workflow to reflect this change?In step `hypershift-azure-create`, `HYPERSHIFT_AZURE_MARKETPLACE_IMAGE_PUBLISHER`,`HYPERSHIFT_AZURE_MARKETPLACE_IMAGE_OFFER`, `HYPERSHIFT_AZURE_MARKETPLACE_IMAGE_SKU`,`HYPERSHIFT_AZURE_MARKETPLACE_IMAGE_VERSION` are default to empty, not adding these env here seem satisfied

Reply:
I add the env: HYPERSHIFT_AZURE_MARKETPLACE_ENABLED in my profile azure-ipi-ovn-hypershift-no-marketplace-guest-f14, so no need to consider HYPERSHIFT_AZURE_MARKETPLACE_IMAGE_xxx default or not.